app/search_builders/oregon_digital/errored_oembed_search_builder.rb

```diff
@@ -6,9 +6,11 @@ class ErroredOembedSearchBuilder < OembedSearchBuilder
     self.default_processor_chain += [:with_errored_oembed]
 
     def with_errored_oembed(solr_params)
-      error_ids = OembedError.all.map(&:document_id).join(' OR ')
+      ids = OembedError.all.map(&:document_id).join(' OR ')
       solr_params[:fq] ||= []
-      solr_params[:fq] = "id:(#{error_ids})"
+      query = solr_params[:fq].length.zero? ? '' : ' AND '
+      filter = ids.blank? ? '-id:*' : "id:(#{ids})"
+      solr_params[:fq] << "#{query}#{filter}"
```

Comment:
`solr_params[:fq]` is supposed to be an array, but comes back as a string from OembedSearchBuilder, similar to Hyrax code: https://github.com/samvera/hyrax/blob/master/app/search_builders/hyrax/embargo_search_builder.rb#L17
This is causing inconsistent behavior between actually running (string) and testing (array)
changing L18 of OembedSearchBuilder to `solr_params[:fq] << 'oembed_url_tesim:*'`, removing `query` here, and updating tests should fix the issue